.gitlab/choco_build/choco_build.yml

```diff
@@ -27,7 +27,7 @@ windows_choco_offline_7_x64:
 # it is run only once the msi package is pushed
 windows_choco_online_7_x64:
   rules:
-    !reference [.on_deploy_stable_or_beta_repo_branch_manual]
+    !reference [.on_deploy_stable_or_beta_repo_branch]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ is this an intentional change, to have it run automatically?

Reply:
Yes, I wanted to test that the datadog-agent part automatically uploads the chocolatey package to S3. Then manually test the agent-release-management part. This way I don't have to make two small PRs for datadog-agent (one to upload and then one to automate it). Do you think it is safer to separate it into two?

Reply:
If the deploy job that depends on it is also manual I don't think there's a risk 👍